tests/acceptance/bootstrap/SpacesContext.php

```diff
@@ -1345,6 +1345,29 @@ public function theUserUploadsAFileToSpace(
 		$this->featureContext->setResponse($response);
 	}
 
+	/**
+	 * @When /^user "([^"]*)" uploads a file inside federated space "([^"]*)" with content "([^"]*)" to "([^"]*)" using the WebDAV API$/
+	 *
+	 * @param string $user
+	 * @param string $spaceName
+	 * @param string $content
+	 * @param string $destination
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 * @throws Exception
+	 */
+	public function theUserUploadsAFileToFedratedSpace(
```

Comment:
```suggestion
	public function theUserUploadsAFileInsideFedratedSpace(
```
Just matching step definition